ngraph/core/include/ngraph/op/gather.hpp

```diff
@@ -52,5 +52,45 @@ namespace ngraph
                                      const HostTensorVector& inputs) const;
             };
         } // namespace v1
+
+        namespace v7
+        {
+            /// \brief Gather slices from axis of params according to indices
+            class NGRAPH_API Gather : public Op
+            {
+            public:
+                NGRAPH_RTTI_DECLARATION;
+                Gather() = default;
+
+                /// \param data The tensor from which slices are gathered
+                /// \param indices Tensor with indexes to gather
+                /// \param axis The tensor is a dimension index to gather data from
+                /// \param batch_dims The number of batch dimension in data and indices tensors
+                Gather(const Output<Node>& data,
+                       const Output<Node>& indices,
+                       const Output<Node>& axis,
+                       const int64_t batch_dims);
+
+                /// \param data The tensor from which slices are gathered
+                /// \param indices Tensor with indexes to gather
+                /// \param axis The tensor is a dimension index to gather data from
+                Gather(const Output<Node>& data,
+                       const Output<Node>& indices,
+                       const Output<Node>& axis);
```

Comment:
May be remove this constructor and update the previous one with the default value for `batch_dims = 0`?

Reply:
Agree, will rewrite this constructor

Reply:
done